lms/static/scripts/frontend_apps/config.js

```diff
@@ -82,31 +90,34 @@ import { createContext } from 'preact';
  */
 
 /**
- * @typedef {'blackboard_missing_integration' |
- *  'canvas_invalid_scope' } OAuthErrorCode
- */
-
-/**
- * Configuration for the error dialog shown if authorizing access to an
- * OAuth API fails.
- *
- * @typedef OAuthErrorConfig
- * @prop {string|null} authUrl
- * @prop {string} errorDetails
- * @prop {OAuthErrorCode|null} errorCode
- * @prop {string[]} canvasScopes
+ * Base type for error information provided through this configuration object.
+ *
+ * @typedef ConfigErrorBase
+ * @prop {'blackboard_missing_integration' | 'canvas_invalid_scope' | 'reused_consumer_key'} [errorCode]
```

Comment:
Just to be consistent with the whitespace compared with other type unions in this file:
```suggestion
 * @prop {'blackboard_missing_integration'|'canvas_invalid_scope'| 'reused_consumer_key'} [errorCode]
```